client/assets/localScripts/SwitchItowns.js

```diff
@@ -121,6 +120,8 @@ module.exports = class SwitchItowns {
         } else {
           //remove avatar controls
           avatarController.setAvatarControllerMode(false, localCtx);
+          const refine = localCtx.findLocalScriptWithID('itowns_refine');
```

Comment:
not sure about this (same in CityMockUp) its quite laggy test it to be sure, otherwise refine at the end of the routine

Reply:
not the case anymore in CityMockUp since there is a custom refine there